src/main/java/uk/gov/digital/ho/hocs/audit/entrypoint/CustomExportResource.java

```diff
@@ -26,41 +28,29 @@ public CustomExportResource(CustomExportService customExportService) {
     }
 
     @GetMapping(value = "/export/custom/{viewName}", produces = "text/csv;charset=UTF-8")
-    public @ResponseBody
-    void getCustomDataExport(HttpServletResponse response,
+    public void getCustomDataExport(HttpServletResponse response,
                              @PathVariable("viewName") String viewName,
                              @RequestParam(name = "convertHeader", defaultValue = "false") boolean convertHeader) {
-        response.setHeader(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + getFilename(viewName));
-
         try {
+            setResponseHeaders(response,getFilename(viewName));
             customExportService.export(response, viewName, convertHeader);
-            response.setStatus(200);
-        } catch (Exception ex) {
-            log.error("Error exporting CSV file for custom report {}: {}", viewName, ex.getMessage());
-            if (ex instanceof HttpClientErrorException) {
-                response.setStatus(((HttpClientErrorException) ex).getRawStatusCode());
-            } else if (ex instanceof EntityPermissionException) {
-                response.setStatus(HttpStatus.FORBIDDEN.value());
-            } else {
-                response.setStatus(HttpStatus.INTERNAL_SERVER_ERROR.value());
-            }
         }
+        catch (IOException e) {
+            throw new ApplicationExceptions.AuditExportException(e, EXPORT_FAILURE_CUSTOM, "Unable to export Custom row for %s", viewName);
+        }
+        response.setStatus(HttpStatus.OK.value());
     }
 
     @PostMapping(value = "/admin/export/custom/{viewName}/refresh")
-    public @ResponseBody
-    void refreshMaterialisedView(HttpServletResponse response, @PathVariable("viewName") String viewName) {
-        try {
-            customExportService.refreshMaterialisedView(viewName);
-            response.setStatus(HttpStatus.OK.value());
-        } catch (Exception ex) {
-            log.error("Error refreshing materialised view {}: {}", viewName, ex.getMessage());
-            response.setStatus(HttpStatus.INTERNAL_SERVER_ERROR.value());
-        }
+    public void refreshMaterialisedView(HttpServletResponse response, @PathVariable("viewName") String viewName) {
+        customExportService.refreshMaterialisedView(viewName);
+        response.setStatus(HttpStatus.OK.value());
```

Comment:
I don't think we even need to do this - I believe it defaults to 200 status code..?